Lib/concurrent/futures/process.py

```diff
@@ -377,13 +380,15 @@ class BrokenProcessPool(RuntimeError):
 
 
 class ProcessPoolExecutor(_base.Executor):
-    def __init__(self, max_workers=None):
+    def __init__(self, max_workers=None, context=None):
         """Initializes a new ProcessPoolExecutor instance.
 
         Args:
             max_workers: The maximum number of processes that can be used to
                 execute the given calls. If None or not given then as many
                 worker processes will be created as the machine has processors.
+            context: A multiprocessing context to launch the workers. This
```

Comment:
As a nit, I think calling this parameter `mp_context` would be a bit more explicit.